docs/features/series.md

```diff
@@ -0,0 +1,205 @@
+# Series
+Series enables customers to bundle episodic content such as TV shows and learning courses or non-episodic content like sports leagues events.  By organizing content into a series, viewers are guided through the content. Series have a predefined sequence of episodes and can be split in seasons.
+
+<img title="" src="img/series.jpg" alt="Series" width="580">
+
+Series are tagged with `Series` in [shelves and libraries](shelves-and-libraries.md):
+
+<img title="" src="img/series-in-library.jpg" alt="Series in library" width="581">
+
+Series are defined through 'series playlist'. This is handled in the first piece of this article.
+
+In the near future JW player will native series construct. This is handled in the second part of this article.
+
+## Series through playlist
+Series are not a native construct in the JW Dashboard at this moment. So customers create a 'series playlist' and set the sequences and episodes using custom parameters.
+
+### Creating series playlists in the dashboard
+The [JW manual](https://support.jwplayer.com/articles/build-an-ott-apps-series-playlist) describes the following process to create a serie playlist: 
+1. Create a Series playlist that automatically adds media as episodes based on series tag, e.g. `seriesId_xdAqW8ya`
+2. Upload the media items and
+   - tag them with the series id. E.g. `seriesId_xdAqW8ya` 
+   - provide it with the episode number e.g. `episodeNumber: 1`
+   - provide it with the season number e.g. `seasonNumber: 1`
+3. Add a media item that points to the series playlist using a parameter, e.g:  `seriesId: xdAqW8ya`. It's used as the card:
+4. Add the card to playlists  used in [shelves and libraries](shelves-and-libraries.md) E.g. library 'All Shows'
+   
+### Series in libraries and shelves
+[Shelves and libraries](shelves-and-libraries.md) load their data using the [GET playlist endpoint](https://developer.jwplayer.com/jwplayer/reference/get_v2-playlists-playlist-id). Some items in this  playlis refer to series. These are identified using the `seriesId` , which links to the  playlist that contains the episodes. 
+   
+   ```
+   GET playlist/o45EkQBf
+   {
+   "title":"All Courses",
+   "description":"",
+   "kind":"DYNAMIC",
+   "feedid":"o45EkQBf",
+   "playlist":[
+    {
+      "mediaid":"txHnOU0x",
+      "seriesId":"xdAqW8ya",
+      "episodes":"4"
+      "title":"Primitive Animals",
+      "image":"https://content.jwplatform.com/v2/media/txHnOU0x/poster.jpg?width=720",
+      "duration":0,
+      "pubdate":1615379806,
+      "description":"Aimed at beginners, this workshop helps you connect basic Blender functionality into a complete workflow and mindset for building characters.",
+      "tags":"course,Beginner"
+    },
+    { }
+   ]
+   }
+   ```
+   
+### Serie detail window
```

Comment:
I can see the term "serie" without thinking it's a spelling mistake.  I think it's not too confusing to use series to mean both singular and plural and rely on context to differentiate.

Reply:
Agreed. This 'serie' is a spelling mistake.